### PR TITLE
Create queue

### PR DESCRIPTION
**Pull request by iamjackslayer**
added enqueue and queue model, not implemented yet.

### CODE REVIEW

**Comment by gingivitiss** [comment]
Code quality is a-ok.

**Comment by iamjackslayer** [comment]
Can merge this PR?

**Comment by jjlee050** [comment]
Can but then u still hav outstanding issues to fix from the checks.

**Comment by iamjackslayer** [comment]
My mistake, I did not push my changes into this PR branch of mine.  #64 fixes are now reflected in this branch.

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -108,17 +110,21 @@ public void updatePerson(Person target, Person editedPerson) {
         versionedAddressBook.updatePerson(target, editedPerson);
         indicateAddressBookChanged();
     }
-
-    //@@author jjlee050
```
[the remark this answers is not in the document]
Comment (iamjackslayer) [reply src/main/java/seedu/address/model/ModelManager.java]: Noted.

File: src/main/java/seedu/address/model/ModelManager.java [hunk 2]
```diff
@@ -103,18 +104,18 @@ public void addDoctor(Doctor doctor) {
         updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
         indicateAddressBookChanged();
     }
-
+    @Override
+    public void enqueue(Person target) {
+        patientQueue.add(target);
+    }
+    //@@author jjlee050
```
Comment (jjlee050) [inline src/main/java/seedu/address/model/ModelManager.java]: This is suppose to be at updateDoctor

**Comment by jjlee050** [comment]
You can put //@@author xxx. (Optional for now). In addition you fail travis build.

**Comment by iamjackslayer** [comment]
I can check the log from the github. But is there a way to run appveyor check on my local machine?

**Comment by jjlee050** [comment]
Refer to https://github.com/orgs/CS2103-AY1819S1-W14-1/teams/developers. Did you try the command locally?

**Comment by iamjackslayer** [comment]
Ok thanks. I can now run gradle locally on terminal.

**Pull request merged**